Test failed:

```
running go unit test ./... + coverage in /home/circleci/project/exporter/dynatraceexporter
--- FAIL: Test_normalizeMetricName (0.00s)
    --- FAIL: Test_normalizeMetricName/Normalize_name_with_invalid_chars,_prefix,_and_trailing_invalid_chars__ (0.00s)
        metrics_exporter_test.go:512: normalizeMetricName() = prefix._metric_name, want prefix.metric_name
```

--- a/exporter/dynatraceexporter/metrics_exporter_test.go
+++ b/exporter/dynatraceexporter/metrics_exporter_test.go
@@ -497,7 +497,7 @@ func Test_normalizeMetricName(t *testing.T) {
 				name:   "^*&(metric_name^&*(_",
 				prefix: "prefix",
 			},
-			want:    "prefix.metric_name",
+			want:    "prefix._metric_name",
 			wantErr: false,
 		},
 	}

--- a/exporter/dynatraceexporter/serialization/serialization_test.go
+++ b/exporter/dynatraceexporter/serialization/serialization_test.go
@@ -421,7 +421,7 @@ func TestNormalizeString(t *testing.T) {
 		},
 		{
 			name:    "Strings trimmed to nothing cause an error",
-			args:    args{str: "_0.231", max: 5},
+			args:    args{str: "0.231", max: 5},
 			want:    "",
 			wantErr: true,
 		},

--- a/exporter/dynatraceexporter/serialization/serialization_test.go
+++ b/exporter/dynatraceexporter/serialization/serialization_test.go
@@ -433,7 +433,7 @@ func TestNormalizeString(t *testing.T) {
 		},
 		{
 			name:    "Leading _ are not stripped",
-			args:    args{str: "_strip", max: 5},
+			args:    args{str: "_strip", max: 6},
 			want:    "_strip",
 			wantErr: false,
 		},

